### PR TITLE
`General`: Add troubleshooting page to the documentation

### PR DESCRIPTION
**Pull request by Mtze**
### Motivation and Context
@dfuchss  encountered a problem while downloading the repositories of a large course - See #4454 

This PR adds a troubleshooting page to the documentation and adds details on how to fix the download problem. 

The Ansible Collection repository has also been updated - The timeouts are now configurable. See https://github.com/ls1intum/artemis-ansible-collection/pull/4

### CODE REVIEW

**Comment by Mtze** [comment]
The release https://github.com/ls1intum/artemis-ansible-collection/releases/tag/v0.1.1 of the ansible collection now allows the configuration of timeouts with ansible variables

File: docs/admin/troubleshooting.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Troubleshooting
+===============
+
+
+Repository download  of large courses fails
+-------------------------------------------
+
+The most likely issue is that the configured timeouts on the reverse proxy are to low.
```
Comment (julian-christl) [inline docs/admin/troubleshooting.rst:8]:
```suggestion
The most likely issue is that the configured timeouts on the reverse proxy are too low. 
```

File: docs/admin/troubleshooting.rst [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+===============
+
+
+Repository download  of large courses fails
+-------------------------------------------
+
+The most likely issue is that the configured timeouts on the reverse proxy are to low. 
+
+If you set up your Artemis instance with the `Ansible Role (>=v0.1.1) <https://github.com/ls1intum/artemis-ansible-collection>`__, 
+you can set the timeout values with the following varaibles:
```
Comment (julian-christl) [inline docs/admin/troubleshooting.rst:11]:
```suggestion
If you set up your Artemis instance with the `Ansible Role (>=v0.1.1) <https://github.com/ls1intum/artemis-ansible-collection>`__, you can set the timeout values with the following variables:
```

**Review by julian-christl — CHANGES_REQUESTED** [review]

**Review by julian-christl — DISMISSED** [review]
Looks good, thanks.

File: docs/admin/troubleshooting.rst [hunk 3]
```diff
@@ -0,0 +1,17 @@
+Troubleshooting
+===============
+
+
+Repository download  of large courses fails
+-------------------------------------------
```
Comment (b-fein) [inline docs/admin/troubleshooting.rst:6]:
```suggestion
Repository download of large courses fails
------------------------------------------
```

File: docs/admin/troubleshooting.rst [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+The most likely issue is that the configured timeouts on the reverse proxy are too low. 
+
+If you set up your Artemis instance with the `Ansible Role (>=v0.1.1) <https://github.com/ls1intum/artemis-ansible-collection>`__, you can set the timeout values with the following variables:
```
Comment (b-fein) [inline docs/admin/troubleshooting.rst:10]: You could also mention here that the same configuration parameters can be used in a manually set up nginx, as I assume this is the most common setup at the moment.

**Review by b-fein — APPROVED** [review]
Looks good.
I think it should be mentioned how this issue can be resolved for more general setups as commented inline.

File: docs/admin/troubleshooting.rst [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+.. code:: yaml
+
+   proxy_send_timeout: "900s"
+   proxy_read_timeout: "900s"
+   fastcgi_send_timeout: "900s"
+   fastcgi_read_timeout: "900s"
+
+If you conrigured the ngnix reverse proxy by hand, you have to adapt the following variables in your nginx configurations:
```
Comment (b-fein) [inline docs/admin/troubleshooting.rst:19]:
```suggestion
If you configured the nginx reverse proxy by hand, you have to adapt the following variables in your nginx configurations:

```

**Review by b-fein — DISMISSED** [review]
Thank you. :+1: 

Just two typos and the list needs an empty empty line beforehand to be formatted properly in the html.

File: docs/admin/troubleshooting.rst (hunk 5 above)
Comment (Mtze) [reply docs/admin/troubleshooting.rst:19]: 🙈

**Comment by Mtze** [comment]
Thank you for your feedback!

**Comment by julian-christl** [comment]
And please adapt the title of the PR according to our guidelines. There is a reason why there is a checklist item for that ;)

**Pull request merged**